vmm/src/vm.rs

```diff
@@ -423,6 +465,9 @@ impl Vcpu {
                     Ok(())
                 }
                 VcpuExit::IoOut(addr, data) => {
+                    if addr == DEBUG_IOPORT && data.len() == 1 {
+                        self.log_debug_ioport(data[0]);
```

Comment:
We should add a `return Ok(());` here, so that we don't even try to write to the bus. This will save some performance I think.